tasks/config/npm/gulpfile.js

```diff
@@ -2,33 +2,33 @@
  *           Set-up
  *******************************/
 
-var
+let
     gulp   = require('gulp'),
 
     // read user config to know what task to load
-    config = require('./tasks/config/user')
+    config = require('./tasks/config/user') // eslint-disable-line import/extensions
```

Comment:
this file is copied in two places using `gulp.src(source.userGulpFile)` - do we really need that instead of the `/gulpfile.js`

please notice `tasks/config/npm/gulpfile.js` vs. `/gulpfile.js` which are almost identical